public_html/composer.json

```diff
@@ -0,0 +1,129 @@
+{
+	"name": "wordcamp/wordcamp.org",
+	"description": "",
+	"homepage": "https://wordcamp.org",
+	"license": "GPL-2.0-or-later",
+	"support": {
+		"issues": "https://github.com/WordPress/wordcamp.org/issues"
+	},
+	"config": {
+		"platform": {
+			"php": "7.2"
+		},
+		"vendor-dir": "wp-content/mu-plugins/vendor"
+	},
+	"repositories": [
+		{
+			"type": "composer",
+			"url": "https://wpackagist.org/"
+		},
+		{
+			"type": "package",
+			"package": [
+				{
+					"name": "wordpress-plugin/bbpress",
+					"type": "wordpress-plugin",
+					"version": "2.6",
+					"source": {
+						"type": "svn",
+						"url": "https://plugins.svn.wordpress.org/bbpress/",
+						"reference": "branches/2.6/"
+					}
+				},
+				{
+					"name": "wordpress-plugin/camptix-bd-payments",
+					"type": "wordpress-plugin",
+					"version": "1.2",
+					"source": {
+						"type": "svn",
+						"url": "https://plugins.svn.wordpress.org/bd-payments-camptix/",
+						"reference": "tags/1.2/"
+					}
+				},
+				{
+					"name": "wordpress-plugin/camptix-paystack",
+					"type": "wordpress-plugin",
+					"version": "1.0.0",
+					"source": {
+						"type": "svn",
+						"url": "https://plugins.svn.wordpress.org/tbz-camptix-paystack/",
+						"reference": "tags/1.0.0/"
+					}
+				},
+				{
+					"name": "wordpress-plugin/camptix-paynow",
+					"type": "wordpress-plugin",
+					"version": "1.0.4",
+					"source": {
+						"type": "svn",
+						"url": "https://plugins.svn.wordpress.org/gateway-camptix-paynow-payment/",
+						"reference": "tags/1.0.4/"
+					}
+				},
+				{
+					"name": "wordpress-plugin/json-rest-api",
+					"type": "wordpress-plugin",
+					"version": "1.2.5",
+					"source": {
+						"type": "svn",
+						"url": "https://plugins.svn.wordpress.org/json-rest-api/",
+						"reference": "tags/1.2.5/"
+					}
+				},
+				{
+					"name": "wordpress-meta/wporg-profiles-wp-activity-notifier",
+					"type": "wordpress-plugin",
+					"version": "1.1",
+					"source": {
+						"type": "svn",
+						"url": "https://meta.svn.wordpress.org/sites/trunk/wordpress.org/public_html/wp-content/plugins/",
+						"reference": "wporg-profiles-wp-activity-notifier/"
+					}
+				}
+			]
+		}
+	],
+	"require": {
+		"league/oauth2-client": "2.4.1"
+	},
+	"require-dev": {
```

Comment:
`SHOULD` - Similar to above, it'd be good to document the reason why these are dev dependencies somewhere/somehow.

Reply:
See https://github.com/WordPress/wordcamp.org/pull/152#discussion_r304109387

Reply:
Maybe in our devdocs then?

Reply:
Done.